src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -187,6 +187,7 @@ Copyright (c) .NET Foundation. All rights reserved.
          AddModules="@(AddModules)"
          AdditionalFiles="@(AdditionalFiles)"
          AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
+         Analyzers="@(Analyzer)"
```

Comment:
@pranavkm couldn't we just remove the ANalyzers from this phase wholesale?

Reply:
Source generators are also considered analyzers, and they need to happen in this phase.

Reply:
Yep, thats the problem, and why we added the new `SkipAnalyzers` argument. 

Equally we're going to have this issue if a generator emits a warning too, so we might need to re-think the strategy a little bit? I wonder if we should just disable all wanings (`-warn 0`) for the component discovery phase? That way any warnings would only show up in the second phase, and errors in the first phase would stop the second running anyway.

Reply:
I'll give bumping up the SDK another try. If we get it updated to the latest and greatest, perhaps we get this change in as-is.

Reply:
> I wonder if we should just disable all wanings (`-warn 0`) for the component discovery phase? That way any warnings would only show up in the second phase, and errors in the first phase would stop the second running anyway.

ya I think that'd be best

Reply:
Ok, I've disabled warnings and removed the `SkipAnalyzers` param. Means that phase might be a little slower until you upgrade the SDK and can use it again, but everything should work as expected.

Reply:
Should we file a follow up issue to move to `SkipAnalyzers` at some point in the future?

Reply:
Yes, I'll open an issue to track.